### DISCUSSION TITLE
True Center Option

### DISCUSSION THREAD

**Discussion by trijste** [rxhanson/Rectangle#1040] [Ideas]
Accounting for Stage Manager makes centring off-center. I think, for aesthetic, there should be a shortcut to center the window truly.

I've tried to manage this with a Shortcut, but it doesn't work as well as Rectangle.

Thanks for Rectangle.

**Comment by rxhanson** [comment]
Thanks for the feedback. Makes sense to me. I’ll see if I can add this in an upcoming release. 

I think it makes the most sense to default center to true center and allow it to be changed back to what it is now via terminal command. I don’t think it’s worth the complexity of adding another center command and having to explain the difference in the ui.

**Reply by trijste** [reply]
I think that makes the most sense too.

**Reply by trijste** [reply]
I just purchased Rectangle Pro and hope this will also roll up to that app. 😻

Thanks, Ryan!

**Reply by rxhanson** [reply]
Thanks! It'll definitely make its way into Pro as well.
